### DISCUSSION TITLE
Changing statistics through events

### DISCUSSION THREAD

**Discussion by genitr** [AscensionGameDev/Intersect-Engine#1506] [Ideas]
**Is your feature request related to a problem? Please describe.**
Only two statistics can be changed through events - Health and Mana.  This is extremely small, because the game character has other statistics as well.

**Describe the solution you'd like**
There is a well-grounded proposal to make it possible to change other statistics.  And I think many will agree with me. 

**Additional context**
With this, many good ideas can be realized.  For example, a condition or a set of armor with improved characteristics (when all the elements are equipped), a system of fatigue, thirst and hunger, at a certain time of the day (night, day, morning) or on a certain map or under other conditions, you can, for example, lower the speed or magic resistance of the character, and so on.
It would be great to see this feature in future engine updates.

**Comment by WeylonSantana** [comment]
The intersect has a kind of tracker for the points, making the events do this could mess up but things, especially if not used correctly, that being said I'm against it. At most we could do something to get around it, like making the events add buffs to the players maybe, but if whoever created it wants the change to be permanent, I think it's too complicated then
